src/PrestaShopBundle/Resources/config/services/bundle/form/form_handler.yml

```diff
@@ -495,22 +495,6 @@ services:
             - 'PrestaShopBundle\Form\Admin\Improve\Design\Theme\ShopLogosType'
             - 'ShopLogo'
 
-    # Entity form handler
-    prestashop.admin.request_sql.form_handler:
```

Comment:
Same question as before, but why remove these services?

Reply:
Service has been marked as deprecated in the previous version.
It's time to remove them now :D

Reply:
Yes but it's not the place of this PR, this PR is about deprecations related to the Symfony upgrade not our own
Our deprecated services/classes should be removed in a dedicated PR
(but I'm not gonna block for this one of course)

Reply:
Same here, I think it was because it was marked as deprecated 😅